docs/book.json

```diff
@@ -0,0 +1,19 @@
+{
+  "plugins": ["edit-link", "prism", "-highlight", "github", "anchorjs"],
+  "pluginsConfig": {
+    "edit-link": {
+      "base": "https://github.com/kube-aws/docs/tree/master",
+      "label": "Edit This Page"
+    },
+    "github": {
+      "url": "https://github.com/kubernetes-incubator/kube-aws"
+    }
+  },
+  "links": {
+    "sidebar": {
+    }
+  },
+  "styles": {
+    "website": "styles/site.css"
+  }
+}
```

Comment:
Can do, it also means moving things like the .gitignore for node_modules into the root as well and the `_book` will be generated in the root. In this PR it's all self contained in the `docs` folder. So if you want to combine them it's possible, are you ok that it will impact these other parts?